packages/magic/magic.0.7.3/opam

```diff
@@ -8,7 +8,10 @@ remove: [
   ["./configure" "--prefix" prefix "--datarootdir" prefix]
   [make "uninstall"]
 ]
-depends: ["ocaml" "ocamlfind"]
+depends: [
+  "ocaml" {< "5.0"}
+  "ocamlfind"
```

Comment:
There seem to be some packaging error on Archlinux where the configure script fails because it can't find `which`: `./configure: line 1708: which: command not found`.

```suggestion
  "ocamlfind"
  "conf-which" {build}
```